resources/js/Pages/Auth/Register.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import Form from "@/Organisms/Form";
+import FormTextInput from "@/Molecules/FormTextInput";
+import AuthLayout from "@/Atoms/AuthLayout";
+
+const RegisterForm: React.FC = () => {
+	return (
+		<AuthLayout
+			heading="Register"
+			message="Please register to create a profile"
+		>
+			<Form
+				action=""
+				initialValues={{
+					fullname: "",
+					email: "",
+					password: "",
+					confirmpwd: "",
+				}}
+				button="Register"
+				render={(values, handleChange, errors) => (
+					<React.Fragment>
+						<FormTextInput
+							type="text"
+							name="fullname"
+							label="Full Name"
+							value={values["fullname"]}
```

Comment:
Personally I find this a bit easier to read as `values.fullname` (and `errors.fullname` below, etc) but if you prefer it with the `["` then that's not a problem - they are equivalent anyway 😄 
```suggestion
							value={values.fullname}
```

Reply:
Agreed! :)